### PR TITLE
[16.0][ADD] project_task_link

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This module adds a smart button to the project form view, linking to the tasks associated with the project.

@qrtl

### CODE REVIEW

**Review by yostashiro — APPROVED** [review]
LGTM. 👍 

image

**Review by rinaldifirdaus — APPROVED** [review]
Functional Test: Works as Expected.

**Review by desdelinux — APPROVED** [review]
Functional review is ok,

but.... 


@AungKoKoLin1997 Can you make a pr odoo with this?

https://github.com/odoo/odoo/pulls

**Comment by sswapnesh** [comment]
Hi @AungKoKoLin1997 

Screenshot (62)


It should use "Name of the tasks" on smart button.

Screenshot (63)

File: project_task_link/views/project_project_views.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                    name="%(project.act_project_project_2_project_task_all)d"
+                    type="action"
+                    class="oe_stat_button"
+                    icon="fa-tasks"
+                >
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value">
+                            <field name="task_count" />
+                        </span>
+                        <span class="o_stat_text">Tasks</span>
```
Comment (sswapnesh) [inline project_task_link/views/project_project_views.xml]:
```suggestion
                        <span class="o_stat_text">
                            <field name="label_tasks" readonly="1"/>
                        </span>
```

Comment (AungKoKoLin1997) [reply project_task_link/views/project_project_views.xml]: Yes. That makes sense to me. Thanks!

**Comment by AungKoKoLin1997** [comment]
> Can you make a pr odoo with this?

@desdelinux I also created PR in odoo repo. https://github.com/odoo/odoo/pull/130976

**Comment by leemannd** [comment]
Hello @AungKoKoLin1997 , can you check why the tests are failing?

**Comment by AungKoKoLin1997** [comment]
> can you check why the tests are failing?

@leemannd Tests are failing from `project_template` module. I think we don't need follow up this in current PR.

**Review by leemannd — APPROVED** [review]

**Pull request merged**